config/locales/ar.yml

```diff
@@ -14,7 +14,6 @@ ar:
       humane_approach_title: أسلوب يعيد الإعتبار للإنسان
       not_a_product_title: إنك إنسان و لست سلعة
       real_conversation_title: مبني لتحقيق تواصل حقيقي
-    find_another_instance: إبحث عن مثيل خادوم آخر
```

Comment:
Why do we need to remove `find_another_instance` translations? Isn't used by new template anymore, or it's not used since a long time?

Reply:
Mostly because the Travis build will complain if there's unused translations.

To give a bit more context, I recently accidentally linked an instance with closed registrations to someone who hasn't heard of Mastodon. They believed Mastodon was closed for sign ups, despite the "Find another instance" button, because, I'm assuming, it's just too different to what they expect.

By renaming that button back to "Sign up" we might do better.